pkg/build/builder/common.go

```diff
@@ -20,7 +22,20 @@ import (
 // client facing libraries should not be using glog
 var glog = utilglog.ToFile(os.Stderr, 2)
 
-const OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
+const (
+	OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
+
+	// containerNamePrefix prefixes the name of containers launched by a build.
+	// We cannot reuse the prefix "k8s" because we don't want the containers to
+	// be managed by a kubelet.
+	containerNamePrefix = "openshift"
+
+	StatusMessagePostCommitHookFailed      = "Failed because of post commit hook"
+	StatusMessagePushImageToRegistryFailed = "Failed pushing the image to the registry"
+	StatusMessagePullBuilderImageFailed    = "Failed pulling builder image"
+	StatusMessageFetchSourceFailed         = "Failed to fetch the input source"
+	StatusMessageInvalidOutputRef          = "No output image specified"
```

Comment:
"Output image could not be resolved".  (it's valid to have no output image specified, it's not valid to specify one we can't resolve)